libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/CancellableHandler.kt

```diff
@@ -0,0 +1,34 @@
+package com.mapbox.navigation.core.routerefresh
+
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+internal class CancellableHandler(
+    private val scope: CoroutineScope
+) {
+
+    private val jobs = linkedSetOf<Job>()
```

Comment:
when could it have many jobs? I guess we always refresh the same set of routes, don't we?

Reply:
Well, I think right now we really can't get multiple jobs here, but this class doesn't know anything about how it's going to be used. For example, it shouldn't know that there is some executor that will filter out the request if we already have a running request.

Reply:
Removed this class after all.